OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -291,6 +291,7 @@ public void onException(Exception ex) {
                         }
                     }
                     if (fieldErrorShown) {
+                        showErrorPopup();
```

Comment:
Nvm. Bug still exists against Prod. 

Oddly, stage has a different issue with the alert popping up twice.